crates/node-db-sql/sql/query/get_block.sql

```diff
@@ -2,10 +2,12 @@ SELECT
     block.number,
     block.timestamp_secs,
     block.timestamp_nanos,
```

Comment:
I think we already have a `get_block_header` which we could use to get these values instead of returning them multiple times here.

Reply:
Although this wasn't introduced in this PR so feel free to just make an issue for it.

Reply:
Opened #156